### PR TITLE
(feat) parallax scroll on playlist

### PR DESCRIPTION
**Pull request by underaj**

### CODE REVIEW

File: src/playlist/Playlist.js [hunk 1]
```diff
@@ -1,35 +1,87 @@
...
+    return (
+      <View key="background" style={s.background}>
+        <Image source={imageSource} />
+        <View style={s.backgroundOverlay} />
+      </View>
+    );
+  }
+
+  render() {
+    const { onScroll = () => {} } = this.props;
```
Comment (dting) [inline src/playlist/Playlist.js]: What does this do?

Comment (underaj) [inline src/playlist/Playlist.js]: It's done on the example code from the NPM module, i used it to make sure it doesn't break.

Comment (dting) [inline src/playlist/Playlist.js]: This is a destructuring assignment with a default. I don't think we need it unless we plan on passing in an onScroll to this component's props to override the ScrollView's onScroll behavior.

See if we can remove the destructuring assignment and the onScroll prop of the ScrollView component.

Comment (underaj) [inline src/playlist/Playlist.js]: Taken out.

**Pull request merged**